### ISSUE TITLE
Restore Diagnostics Meeting usual frequency

### ISSUE THREAD

**Issue by RafaelGSS** [nodejs/diagnostics#581]
Since https://github.com/nodejs/diagnostics/issues/507.

We've had no meeting in the second timezone (10.30 PM UTC) so far, which means that we are having a single meeting per month. I'd like suggest to restore all the meetings to the original timing (example: https://github.com/nodejs/diagnostics/issues/574).

We're getting good attendance and might make sense to call more frequently (every 2 weeks at the same time)

**Comment by Qard** [comment]
Yeah, sorry about that. I had planned on taking the other time slot and then moved to another timezone for a few months where that's the middle of the night. 😅

**Comment by RafaelGSS** [comment]
That's fine! The first timezone is now good for you?

**Comment by Qard** [comment]
For now...I'll be on the move again in September, so we'll see how it goes. I'd just not worry too much about accommodating me for now. 😅

**Comment by tony-go** [comment]
+1

**Comment by RafaelGSS** [comment]
ping @nodejs/diagnostics

**Comment by tony-go** [comment]
Let's have a chat on this one at the next meeting.

**Comment by Qard** [comment]
The next one is another weirdly-timed one, so maybe just decide here? I see no objections though and this has been open for awhile, so maybe just restore it?

**Comment by RafaelGSS** [comment]
@mhdawson could you do it for us?

**Comment by mhdawson** [comment]
@RafaelGSS, since the issue is already open for this week, I changed it for the next one. I believe I moved it to the righ time. Can you take a look and confirm?

**Comment by RafaelGSS** [comment]
All good!
